packages/flutter/lib/src/cupertino/bottom_tab_bar.dart

```diff
@@ -176,13 +182,10 @@ class CupertinoTabBar extends StatelessWidget implements PreferredSizeWidget {
           data: IconThemeData(color: inactive, size: iconSize),
           child: DefaultTextStyle( // Default with the inactive state.
             style: CupertinoTheme.of(context).textTheme.tabLabelTextStyle.copyWith(color: inactive),
-            child: Padding(
```

Comment:
Why is the `Padding` removed here?

Reply:
bc. the padding of the item itself needs to be inside the Row, not outside, so each item has the padding, not the row itself.
I added the padding in line 227